client/my-sites/stats/activity-log/index.jsx

```diff
@@ -536,24 +527,58 @@ class ActivityLog extends Component {
 
 		const restoreConfirmDialog = requestedRestoreActivity && (
 			<ActivityLogConfirmDialog
-				applySiteOffset={ this.applySiteOffset }
 				key="activity-rewind-dialog"
-				onClose={ this.handleCloseDialog }
-				onConfirm={ this.handleConfirmDialog }
-				timestamp={ requestedRestoreActivity.activityTs }
-			/>
+				confirmTitle={ translate( 'Confirm Rewind' ) }
+				onClose={ this.dismissRestore }
+				onConfirm={ this.confirmRestore }
+				title={ translate( 'Rewind Site' ) }
+			>
+				{ translate(
+					'This is the selected point for your site Rewind. ' +
+						'Are you sure you want to rewind your site back to {{b}}%(time)s{{/b}}?',
+					{
+						args: {
+							time: this.applySiteOffset(
+								moment.utc( requestedRestoreActivity.activityTs )
+							).format( 'LLL' ),
+						},
+						components: { b: <b /> },
+					}
+				) }
+				<div className="activity-log-confirm-dialog__notice">
+					<Gridicon icon={ 'notice' } />
+					<span className="activity-log-confirm-dialog__notice-content">
+						{ translate(
+							'This will remove all content and options created or changed since then.'
+						) }
+					</span>
+				</div>
+			</ActivityLogConfirmDialog>
 		);
 
 		const backupConfirmDialog = requestedBackup && (
 			<ActivityLogConfirmDialog
-				applySiteOffset={ this.applySiteOffset }
 				key="activity-backup-dialog"
-				onClose={ this.handleCloseDialog }
-				onConfirm={ this.handleConfirmDialog }
-				timestamp={ requestedBackup.activityTs }
+				confirmTitle={ translate( 'Create download' ) }
+				onClose={ this.dismissBackup }
+				onConfirm={ this.confirmDownload }
+				title={ translate( 'Create downloadable backup' ) }
 				type={ 'backup' }
 				icon={ 'cloud-download' }
-			/>
+			>
+				{ translate(
+					'We will build a downloadable backup of your site at {{b}}%(time)s{{/b}}. ' +
+						'You will get a notification when the backup is ready to download.',
+					{
+						args: {
+							time: this.applySiteOffset( moment.utc( requestedBackup.activityTs ) ).format(
+								'LLL'
+							),
+						},
+						components: { b: <b /> },
+					}
+				) }
```

Comment:
Same than above, replace args with only components.

Reply:
changed in 3e5c68d41